components/breadcrumb-nav/css/_mixin.scss

```diff
@@ -55,6 +62,7 @@
 
 @mixin utrecht-breadcrumb-nav__link--current {
   --utrecht-link-current-font-weight: var(--utrecht-breadcrumb-nav-link-current-font-weight, inherit);
+  --utrecht-link-hover-text-decoration: none;
```

Comment:
Misschien is deze style niet nodig. Je kunt `current` samen met `disabled` gebruiken, en `disabled` links hebben sowieso geen interactieve states zoals hover.